locales/en-US/messages.properties

```diff
@@ -102,6 +102,13 @@ denotes_required_field=denotes required field
 share_a_link_with_your_fellowers=Share a link with your followers
 i_donated_to_mozilla=I donated to @mozilla today because I #lovetheweb. Join me and help build + protect the Web forever
 we_include_you=We want to include you in our work!
+These are the variants:
```

Comment:
This is broken, it should be a comment. As it is, it's interpreted as an empty en-US string.